model/map-jpa/src/main/java/org/keycloak/models/map/storage/jpa/client/entity/JpaClientEntity.java

```diff
@@ -54,6 +55,10 @@
     @Column
     private UUID id;
 
+    //used for implicit optimistic locking
+    @Version
+    private int version;
```

Comment:
Should there be a `@Column` annotation?

Reply:
The `@Column` annotation is optional from a technical point of view as long as you don't use it to change the database field name, type or to add constraints. Basically every time we use the "plain" annotation without any parameters we could leave it out.